README_libf3d.md

```diff
@@ -0,0 +1,218 @@
+# libf3d - A library to render 3D meshes
+
+By Michael Migliore and Mathieu Westphal.
+
+libf3d is a BSD-licensed C++ library to open and render 3D meshes. It is of course used by F3D.
+libf3d API is simple and easy to learn. Python bindings are provided through pybind11.
+
+# Getting Started
+
+Rendering a file and starting the interaction is very easy:
+
+```
+// Create a f3d::engine
+f3d::engine eng();
+
+// Add a file to the loader and load it
+eng.getLoader().addFile("path/to/file.ext").loadFile();
+
+// Start rendering and interacting
+eng.getInteractor().start();
+```
+
+Manipulating the window directly can be done this way:
+
+```
+// Create a f3d::engine
+f3d::engine eng();
+
+// Add a file to the loader and load it
+eng.getLoader().addFile("path/to/file.ext").loadFile();
+
+// Modify the window and render it
+f3d::window& win = eng.getWindow();
+
+// Set the window size
+win.SetSize(300, 300);
```

Comment:
```suggestion
win.setSize(300, 300);
```

Reply:
And use eng.getLoader() directly

Reply:
eng.getWindow ? but we need it just below.

Reply:
I made a change but Im not sure this is the best way to introduce it.